backend/cmd/analytics/dev.go

```diff
@@ -0,0 +1,17 @@
+//go:build dev
+
+package main
+
+import (
+	"log"
+
+	"github.com/joho/godotenv"
+)
+
+func initLocal() {
```

Comment:
What do you think if we'll make this method as a part of the config reader module? So, it will be reachable for all golang services. No need to keep these dev/test related logic next to main file.